src/Sylius/Behat/Context/Api/Admin/ManagingShipmentsContext.php

```diff
@@ -211,6 +227,28 @@ public function iShouldNotSeeShipmentWithOrderNumber(OrderInterface $order): voi
         );
     }
 
+    /**
+     * @Then I should see :amount :product units in the list
+     */
+    public function iShouldSeeUnitsInTheList(int $amount, ProductInterface $product): void
+    {
+        $shipmentUnitsFromResponse = $this->responseChecker->getResponseContent($this->client->getLastResponse())['units'];
```

Comment:
Why didn't you use:
```suggestion
        $shipmentUnitsFromResponse = $this->responseChecker->getValue($this->client->getLastResponse(), 'units');
```
?